app/src/main/java/org/fossasia/susi/ai/skills/feedback/adapters/recycleradapters/AllReviewsAdapter.kt

```diff
@@ -39,7 +39,10 @@ class AllReviewsAdapter(val context: Context, val feedbackList: List<Feedback>?)
                 if (feedbackList[position].email != null &&
                         !TextUtils.isEmpty(feedbackList[position].email)) {
                     holder.initials.text = feedbackList[position].email?.substring(0, 2)
-                    holder.feedbackEmail?.text = feedbackList[position].email
+                    val truncateAt = feedbackList.get(position).email?.indexOf('@')
+                    if (truncateAt != null && truncateAt is Int) {
+                        holder.feedbackEmail?.text = feedbackList.get(position).email?.substring(0, truncateAt.plus(1)) + " ..."
+                    }
```

Comment:
Add the method in Utils as this is a helper method reused in both files

Duplication is bad

Also, add tests